Wrote section for Ready to Drive Sound

segerpeter07 opened the pull request.

segerpeter07 committed:
Did they do

segerpeter07 committed:
Changed Messages

segerpeter07 committed:
Wrote the ESF Section of the RTD sound

aehoppe commented on esf_main.tex:
```diff
@@ -5,9 +5,63 @@
+
+
+\begin{figure}[h]
+	\includegraphics[width=\linewidth]{Schematic}
```
Please use a more descriptive file name. This file name will create conflicts down the line.

aehoppe commented on esf_main.tex:
```diff
@@ -5,9 +5,63 @@
+		P=5.25W
+	\end{equation}
+%End Equations
+According to these equations, a 1.2K$\Omega$ resistor will function as a current limiting resistor.
```
It'll have to be like a 10W power resistor as well. You should talk about where you're going to mount it to ensure that it gets adequate cooling.

aehoppe commented on esf_main.tex:
```diff
@@ -5,9 +5,63 @@
+%Provide CAD-renderings showing all relevant parts. Mark the parts in the rendering, if necessary.
+The ready to drive sound will be located in the dashboard enclosure shown in Figure !!!!. The buzzer will be mounted
+to the exterior and underside of this enclosure. It must be contained outside of the box so that the buzzer is
+loud enough and underneath to assist with water proofing of the container
```
Is the buzzer waterproof?

segerpeter07 replied:
The new one I chose is 100% waterproof

aehoppe reviewed (changes requested):
Please see my comments for a few changes I'd recommend. However, I think we'll have to redesign this entirely. There will not be TS wiring going into the dashboard. Let's set up a meeting to talk about this.

segerpeter07 committed:
Reworked the RTDS

Changed the circuit to use an N-FET instead of relay, calculated volume
of newly speced siren at 2m, and rewrote necessary sections of
description.

aehoppe commented on esf_main.tex:
```diff
@@ -13,55 +13,28 @@
-resistor, and the buzzer for two seconds. The resistor limits the voltage over the buzzer to 48V
-and the current to 20 mA.
+The Ready to Drive sound is located as a component of the dashboard subsystem, and contains a buzzer (\href{http://www.puiaudio.com/pdf/XL-5530L-LW300-S-R.pdf}{Link: PUI Audio Inc. XL-5530L-LW300-S-R}). The buzzer automatically makes a noise when given power, with the loudness proportional
+to the voltage. The last step in the start up sequence will notify the CAN system that it is time for the ready
```
This should say that the BMS will notify the Dashboard CAN node when the shutdown circuit is complete.

aehoppe commented on esf_main.tex:
```diff
@@ -13,55 +13,28 @@
+	\includegraphics[width=\linewidth]{FormulasForDistanceAndSoundLevel}
+	\caption{Inverse Square Law equations needed to calculate the volume at a certain distance}
+\end{figure}
+Using the inverse square law for sound pressure levels, 123db at 10cm will translate to 117db at 2m which is well over the minimum SPL. 
```
I'm not so thrilled with the volume of this buzzer, it's well into the territory where hearing it at close range will be dangerous to hearing. Can you find one a little quieter, like 100 dB?

aehoppe commented on esf_main.tex:
```diff
@@ -13,55 +13,28 @@
-to the exterior and underside of this enclosure. It must be contained outside of the box so that the buzzer is
-loud enough and underneath to assist with water proofing of the container
+to the exterior and front side of this enclosure. It must be contained outside of the box so that the buzzer is
+loud enough and is waterproof, so it doesn't require any extra protection besides a waterproof O-ring around the mounting point.
```
We'll probably end up using some waterproof silicone sealant like RTV for this for a more permanent seal once we're confident

aehoppe reviewed (changes requested):
Your buzzer should be on the high side of the N-FET so that the FET changes the buzzer's connection to ground, not power.

segerpeter07 committed:
Fixes

Moved the buzzer, speced new buzzer

aehoppe commented on esf_main.tex:
```diff
@@ -13,14 +13,13 @@
-The Ready to Drive sound is located as a component of the dashboard subsystem, and contains a buzzer (\href{http://www.puiaudio.com/pdf/XL-5530L-LW300-S-R.pdf}{Link: PUI Audio Inc. XL-5530L-LW300-S-R}). The buzzer automatically makes a noise when given power, with the loudness proportional
-to the voltage. The last step in the start up sequence will notify the CAN system that it is time for the ready
-to drive sound. This will then provide voltage to the gate of an N-FET which will provide 12V to the buzzer.
+The Ready to Drive sound is located as a component of the dashboard subsystem, and contains a buzzer (\href{http://www.mallory-sonalert.com/specifications/STA20502.PDF}{Link: Mallory Sonalert Products Inc. STA20502}). The buzzer automatically makes a noise when given power, with the loudness proportional
```
Please provide a reference with \ref to a subsection of the appendix that shows a piece of the data sheet with volume charcteristics, and also has a hyperlink to the full datasheet, as per last year's ESF.

aehoppe commented on esf_main.tex:
```diff
@@ -31,10 +30,10 @@ \subsubsection{Wiring, cables, current calculations, connectors}
 	\caption{Inverse Square Law equations needed to calculate the volume at a certain distance}
 \end{figure}
-Using the inverse square law for sound pressure levels, 123db at 10cm will translate to 117db at 2m which is well over the minimum SPL. 
+Using the inverse square law for sound pressure levels, 97db at 122cm will translate to 93db at 2m which is a loud enough volume without inducing harm to the listener. 
```
Also link to the appendix

segerpeter07 committed:
Added Appendix Section

Referenced Appendix section within document

aehoppe commented on esf_main.tex:
```diff
@@ -30,10 +30,20 @@ \subsubsection{Wiring, cables, current calculations, connectors}
+
+
+\newpage
+\subsection{Ready to Drive (APPENDIX SECTION)}\label{Ready to Drive}
```
Can you put this as a subsection after Section 11 (Appendix)? Sorry to keep running this back and forth.

aehoppe reviewed (changes requested):
Just one more thing, please put the appendix section under Section 11 with the same subsection name as the section of the ESF as per last year's document.

segerpeter07 committed:
Changed Section Headers for Appendix Sections

I wasn't able to figure out the paragraph subsections, but they should
be correct when the whole document is compiled.

segerpeter07 committed:
Changed Labels and added Dashboard Rendering and references
